components/plaid/plaid.js

```diff
@@ -19,19 +19,49 @@ const configuration = new Configuration({
   
 const client = new PlaidApi(configuration);
 
+/*
+
+    Assets = "assets",
+    Auth = "auth",
+    Balance = "balance",
+    Identity = "identity",
+    Investments = "investments",
+    Liabilities = "liabilities",
+    PaymentInitiation = "payment_initiation",
+    IdentityVerification = "identity_verification",
+    Transactions = "transactions",
+    CreditDetails = "credit_details",
+    Income = "income",
+    IncomeVerification = "income_verification",
+    DepositSwitch = "deposit_switch",
+    StandingOrders = "standing_orders",
+    Transfer = "transfer",
+    Employment = "employment",
+    RecurringTransactions = "recurring_transactions"

```

Comment:
Is this just for reference? 
We could make makeshift enums out of these if it would be useful

Reply:
Just for reference, the enums are actually within the plaid API itself